ROADMAP.md

```diff
@@ -158,4 +158,78 @@ The following features are under design review:
 * Ability to turn off the self-serve mode, as in many environments there are mechanisms other than the Kubeflow Dashboard that provision/share an environment for/with the user. (#4942)
 * Multi-User Authorization: Add support for K8s RBAC via SubjectAccessReview [#3513](https://github.com/kubeflow/pipelines/issues/3513)
 
-The 1.1 features are tracked in this [Kanban board](https://github.com/orgs/kubeflow/projects/36) 
\ No newline at end of file
+The 1.1 features are tracked in this [Kanban board](https://github.com/orgs/kubeflow/projects/36) 
+
+## Kubeflow 1.2 Features, Release Date: November 2020
+
+Kubeflow 1.2 provides valuable enhancements to HyperParameter Tuning, Pipelines, KFServing, Notebooks and the Training Operators, which improve Kubeflow operations and data scientist productivity.
+
+1.2 includes the following features:
+
+* Katib 0.10 with the new v1beta1 API 
+* Katib support for early stopping.
+* Katib support for custom CRD in the new Trial template.
+* Katib support to resume experiments
+* Katib support for multiple ways to extract metrics
+* KFServing support to add batcher module as sidecar 
+* KFServing for the Alibi explainer upgrade to 0.4.0 
+* KFServing for Triton inference server rename and integrations  
+* Pipelines support for a Tekton backend option.
+* Kubeflow Pipelines 1.0.4, Changelog includes ~20 fixes and ~5 minor features.
+* Notebooks support for Affinity/Toleration configs 
+* Update mxnet-operator manifest to v1
+* Correct XGBoostJob CRD group name and add singular name
+* Fix XGBoost Operator manifest issue
+* Move Pytorch operator e2e tests to AWS Prow
+* Support BytePS in MXNet Operator
+* Fix error when conditions is empty in tf-operator
+* Fix success Policy logic in MXNet Operator
+
+For more details please see this post: https://blog.kubeflow.org/release/official/2020/11/18/kubeflow-1.2-blog-post.html
+
+## Kubeflow 1.3 Features, Target release: March 2021
+
+The Kubeflow 1.3 roadmap includes many User Interface (UI) improvements and core Kubeflow component upgrades to improve installation, management, and authentication.  It also includes support the latest Istio versions.
+
+The 1.3 release plan includes the following features:
+
+User Interface (UI)  enhancements to improve user experience and simplify workflows
+
+* Completely new UIs for KFServing, Katib, Tensorboard & Volumes Manager
+* Notebooks
+  * Important backend updates to Notebooks (i.e. to improve interop with Tensorboard)
+  * New and expanded Jupyter Notebook stack along with easy to customize common base images
+  * Addition of R-Studio and Code-Server (VS-Code) support
```

Comment:
We should either make it explicit that these are best effort ore remove this entry

Reply:
[ I misclicked. Meant the line above for the jupyter notebook stack ]